fix: trap tasks are scored correctly instead of being removed after first scoring

daemontplx opened the pull request:
the bug is: trap key will be deleted after the first scoring, and will never be scored as trap task in subsequent scoring in the rolling-window method

fix: set a TTL to the key that 2X of the rolling window period

jarvis8x7b commented on .env.miner.example:
```diff
@@ -48,3 +48,6 @@ S3_PUBLIC_URL=
 JWT_SECRET=
 # e.g. infura, alchemy API url for ethereum
 ETHEREUM_NODE=
+
```
can we shift this to the top? this section is meant for decentralized env vars

jarvis8x7b commented on dojo/utils/uids.py:
```diff
@@ -29,6 +29,7 @@ def is_miner(metagraph: bt.metagraph, uid: int) -> bool:
     stakes = metagraph.S.tolist()
     from dojo import VALIDATOR_MIN_STAKE
 
+    print(f"Validator min stake {VALIDATOR_MIN_STAKE}")
```
use bt logging

codebender37 replied:
oh, sorry forget to remove

karootplx reviewed (approved):
lgtm

jarvis8x7b commented on .env.miner.example:
```diff
@@ -17,6 +17,7 @@ SUBTENSOR_ENDPOINT=wss://entrypoint-finney.opentensor.ai:443
 # NETUID=98
 # SUBTENSOR_NETWORK=test
 # SUBTENSOR_ENDPOINT=ws://testnet-lite:9944
+# VALIDATOR_MIN_STAKE=1
```
set this to 20k so that if the miner forgets it's set properly

jarvis8x7b commented on README.md:
```diff
@@ -278,6 +278,7 @@ DOJO_API_KEY= # blank for now
 WALLET_COLDKEY=# the name of the coldkey
 WALLET_HOTKEY=# the name of the hotkey
 AXON_PORT=8888 # port to serve requests over the public network for validators to call
+VALIDATOR_MIN_STAKE= # minimum stake required for validators default is 20000 TAO (use this to bypass the blacklist function in testnet)
```
can set to the default 20k value

daemontplx committed:
fix: trap tasks are scored correctly instead of being removed after first scoring

the bug is: trap key will be deleted after the first scoring, and will
never be scored as trap task in subsequent scoring in the rolling-window
method

daemontplx committed:
chore: ttl will be 2x the scoring window
